src/components/features/SaleBox/SaleBox.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './SaleBox.module.scss';
+
+const SaleBox = sale_img => {
+  return (
+    <div>
+      <div className='container'>
+        <div className={styles.wrapper}>
+          <div className={styles.leftinner}>
+            <div className={styles.box_img}>
+              <img src='./images/promotion-bed.jpg' alt='promotion-left' />
+            </div>
+            <p className={styles.promo_text}>
+              GUEST ROOM<span> SOFA</span>
+            </p>
+            <p className={styles.text_price}>-20%</p>
+          </div>
+          <div className={styles.rightinner}>
+            <div>
+              <img src='./images/furnitures-chairs.jpg' alt='promotion' />
```

Comment:
zdjęcia są nadal rozjechane do tego nie widać w ogóle co tam jest napisane. Trzeba albo dobrać odpowiednie zdjęcia z jednolitym tłem, albo  może dodać overlay aby to było widać.

Reply:
Poprawione